docs/swagger.yaml

```diff
@@ -150,7 +150,7 @@ paths:
                         quantity: '2.5'
                         quantityUnit: KG
                       discount: 0
-                      productUrl: 'https://www.vipps.no/store/socks'
+                      productUrl: 'https://vipps.kunde-service.no/vipps-pride-sokker-pakke-a-2-par.html'
```

Comment:
How about `https://example.com/store/socks`  instead?   Then, we don't rely on the kunde-service site and it's clear that it's an example